src/main/clm_initializeMod.F90

```diff
@@ -616,10 +614,8 @@ subroutine initialize2( )
 
     ! use_fates_spitfire is assigned an integer value in the namelist
     ! see bld/namelist_files/namelist_definition_clm4_5.xml for details
-    if (use_fates_spitfire > 1) then
-       call fates_fire_data_inst%initAccVars(bounds_proc)
-    else
-       call fates_fire_no_data_inst%initAccVars(bounds_proc)
+    if (use_fates_spitfire > 0) then
```

Comment:
See note below on the if statement.

Reply:
Ok, I'm keeping the acc interfaces in both the FATESFireNoData and FATESfireData classes (empty in FATESFireNoData) and removing the if statement.